flyteidl/protos/flyteidl/core/literals.proto

```diff
@@ -111,12 +115,18 @@ message Literal {
 
     // Additional metadata for literals.
     map<string, string> metadata = 5;
+}
+
+// A message that contains the metadata of the offloaded data.
+message LiteralOffloadedMetadata {
+    // The location of the offloaded data.
```

Comment:
I think to @wild-endeavor's point, mind adding a note about what we offload here (`core.Literal`)

Reply:
Done